### PR TITLE
UICHKIN-280: Add Jest/RTL tests for `ClaimedReturnedModal` component

### PR DESCRIPTION
**Pull request by Stanislau-Snapok**
## Purpose
Add Jest/RTL testing for `ClaimedReturnedModal` component in `src\components\ClaimedReturnedModal`.

## Refs
https://issues.folio.org/browse/UICHKIN-280

## Screenshots
image

### CODE REVIEW

File: src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Modal,
+  Button,
+} from '@folio/stripes/components';
+
+import ClaimedReturnedModal from './ClaimedReturnedModal';
+
+import { getById } from '../../../test/jest/helpers/utils';
```
Comment (artem-blazhko) [inline src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js:20]:
```suggestion
import { getById } from '../../../test/jest/helpers';
```

File: src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  within,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Modal,
+  Button,
```
Comment (artem-blazhko) [inline src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js:15]: Remove it. It is not used in the code

File: src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  fireEvent,
+  within,
```
Comment (artem-blazhko) [inline src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js:6]: Remove it, it is not used

File: src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js (hunk 1 above)
Comment (Stanislau-Snapok) [reply src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js:20]: Auto import...) Thanks, done!

File: src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js (hunk 2 above)
Comment (Stanislau-Snapok) [reply src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js:15]: You are right! Done.

File: src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js (hunk 3 above)
Comment (Stanislau-Snapok) [reply src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js:6]: Done.

**Review by artem-blazhko — APPROVED** [review]

File: src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+} from '@folio/stripes/components';
+
+import ClaimedReturnedModal from './ClaimedReturnedModal';
+
+import { getById } from '../../../test/jest/helpers';
+
+import { claimedReturnedResolutions } from '../../consts';
+
+FormattedMessage.mockImplementation(({ id, values }) => {
+  const renderValues = (currentValues) => (
```
Comment (pavelspichonak) [inline src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js:23]: nit: { title, barcode, materialType }

Comment (pavelspichonak) [reply src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js:23]: and let us discuss this mock

File: src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+  it('should execute correct function on "returned" button click', () => {
+    expect(onConfirm).not.toBeCalled();
+
+    fireEvent.click(screen.getByTestId(testIds.returnedButton));
+
+    expect(onConfirm).toHaveBeenCalledWith(claimedReturnedResolutions.RETURNED);
+  });
+
+  it('should correctly render "message"', () => {
+    expect(screen.getByText(new RegExp(labelIds.message))).toBeInTheDocument();
```
Comment (pavelspichonak) [inline src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js:135]: I believe it is better to check one full string. let us discuss

**Review by pavelspichonak — APPROVED** [review]

File: src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js (hunk 5 above)
Comment (Stanislau-Snapok) [reply src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js]: discussed

File: src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js (hunk 4 above)
Comment (Stanislau-Snapok) [reply src/components/ClaimedReturnedModal/ClaimedReturnedModal.test.js]: refactored

**Pull request merged**